docs/user/get-started.md

```diff
@@ -0,0 +1,147 @@
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is distibuted high scalable & high-speed approximate nearest neighor seardch engine.<br>
+It provides search result lists for input (e.g. word, sentence, image ...) which is coverted as (also high-dimensional) vector when searching neighor.<br>
+Vald is designed as Cloud Native and also use World Fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) for search neighors.
+
+### Main Features
+
+- Auto Indexing
+    - ???
+
+- Ingress Filltering
+    - ???
+
+- Horizonal Scaling
+    - Vald, which is designed as Cloud Native search engine, can use cluster resource especially memory and cpu.
+
+- Auto Index Backup
+    - Vald has auto index backup with MySQL + Redis or Cassndora which is helpful for quick recovery.
+
+- Distributed Indexing
+    - Vald indexes insert vecotr to distibuted multiple NGT agent. 
+
+- Index Replication
+    - ???
+
+- Easy to use
+    - Vald can be easily installed in a few steps and accepts many kind of settings.
+
+### Requirements
+
+- k8s: 
+- go:
+- docker:
+- helm: 
+
+If helm is not installed, let see below details or [here](https://github.com/helm/helm#install).
+
+<details>
+    <summary>optional installation</summray>
+    <div>
+    <pre>
+        1. helm
+            <code>
+            # brew|Scoop|GoFish
+            [brew|scoop|gofish] install helm
+            # chocolatey
+            choco install kubernetes-helm
+            </code>
+    </pre>
+    </div>
+</details>
+
+## Starting Vald on k8s cluster
+
+You have to install helm. Please check [here](https://github.com/helm/helm#install).
+
+### Deploy
+
+There shows procudure of deployment for vald with scylla, which is used for backup database.
+If you want to learn about scylla more, please refer to [here](https://www.scylladb.com/).
+Please confirm `kubectl cluster-info` before deployment.
+
+1. Prepare scylla database and k8s metrics-server
+
+    At first, you have to apply scylla as backup database.
+
+    ```bash
+    cd {vald_project_root}
+    
+    kubectl apply -f k8s/jobs/db/casandora/configmap.yaml
+    kubectl apply -f k8s/external/scylla
+    ```
+
+    Apply k8s metrics-server
+
+    ```bash
+    kubectl apply -f k8s/metrics/metrics-server
+    ```
+
+2. Deployment from helm template command
+
+    ```
+    helm install --values example/helm/values-scylla.yaml vald
+    ```
+
+### Run
+
+This chapter shows procudure of running vald from localhost with fashion-mnist dataset.
+
+1. Port Forward
+
+    ```bash
+    kubectl port-forward {vald-gateway-pod-name} 8081:8081
+    ```
+
+2. Install dataset
+
+    To search neighors of input vector, vald should have dataset.
+    In this case, you use [fashion-mnist](https://github.com/erikbern/ann-benchmarks) as dataset.
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
the url should be https://github.com/zalandoresearch/fashion-mnist, i think.